charts/tracetest-core/templates/configmap.yaml

```diff
@@ -20,23 +20,28 @@ data:
         samplingRate: {{ .Values.config.profiler.samplingRate }}
         endpoint: {{ .Values.config.profiler.endpoint }}
         environment: {{ .Values.config.profiler.environment }}
-      # exporters:
-      #   collector:
-      #     exporter:
-      #       collector:
-      #         endpoint: collector-opentelemetry-collector.otel:4318
-      #       type: collector
-      #     metricsReaderInterval: 60s
-      #     batchTimeout: 5s
-      #     sampling: 50
-      #     serviceName: tracetest
+      {{- if .Values.config.telemetry.enabled }}
+      exporters:
+        collector:
+          exporter:
+            collector:
+              endpoint: {{ .Values.config.telemetry.collector.endpoint }}
+            type: collector
+          metricsReaderInterval: {{ .Values.config.telemetry.metricsReaderInterval }}
+          batchTimeout: {{ .Values.config.telemetry.batchTimeout }}
+          sampling: {{ .Values.config.telemetry.sampling }}
+          serviceName: tracetest
+          serviceVersion: {{ .Values.image.tag }}
+      {{- end }}
     server:
       httpPort: 11633
-      # telemetry:
-      #   exporter: collector
+      {{- if .Values.config.telemetry.enabled }}
+      telemetry:
+        exporter: collector
+      {{- end }}
 
   provisioning.yaml: |-
       ---
       type: Config
       spec:
-        analyticsEnabled: false
```

Comment:
Does this mean that we are not getting analytics?

Reply:
Nee I don't think we control analytics with this, right?

Reply:
No, use use another flag.

Reply:
No, we use another flag.